syntax = "proto3";

message AddArg {
  int64 a = 1;
  int64 b = 2;
}
message AddReply {
  int64 sum = 1;
}

message SayHelloArg {
  string name = 1;
}
message SayHelloReply {
  string message = 1;
}

service XzxzServiceTest{
  rpc Add(AddArg) returns (AddReply) {}
  rpc SayHello(SayHelloArg) returns (SayHelloReply) {}
}